web/static/js/components/idea_board.jsx

```diff
@@ -9,10 +9,9 @@ import STAGES from "../configs/stages"
 const { ACTION_ITEMS, CLOSED } = STAGES
 
 const IdeaBoard = props => {
-  const { stage } = props
-  const categories = ["happy", "sad", "confused"]
+  const { stage, categories } = props
   const showActionItem = [ACTION_ITEMS, CLOSED].includes(stage)
-  if (showActionItem) { categories.push("action-item") }
+  if (showActionItem && !categories.includes("action-item")) { categories.push("action-item") }
```

Comment:
Hey @tnewell5. Rather than adding a check to see if we've already pushed the new category into the collection. let's just create a constant like `renderableColumnCategories = [...categories, "action-item"]`. Reason being, we don't want to mutate a constant, which `Array.prototype.push` does.

Reply:
cool - thanks!